presidio-analyzer/presidio_analyzer/__init__.py

```diff
@@ -16,9 +16,10 @@
 from presidio_analyzer.recognizer_registry.recognizer_registry \
     import RecognizerRegistry
 from presidio_analyzer.analyzer_engine import AnalyzerEngine
-
+from presidio_analyzer.recognizer_registry.recognizer_registry \
+    import RecognizerStoreApi
 
 __all__ = ['PresidioLogger', 'AnalysisExplanation', 'Pattern',
            'EntityRecognizer', 'LocalRecognizer', 'RecognizerResult',
            'PatternRecognizer', 'RemoteRecognizer', 'RecognizerRegistry',
-           'AnalyzerEngine']
+           'AnalyzerEngine', 'RecognizerStoreApi']
```

Comment:
We don't necessarily want to expose the `ReconizerStoreApi`, it's an internal thing. Everything defined here is meant to be used by the end user.

Reply:
I got a warning from PyCharm that it should be there. I will remove it

Reply:
You should also remove the line which calls it, then you won't have the warning